### PR TITLE
Components

### PR DESCRIPTION
**Pull request by vamsimudadla**

### CODE REVIEW

File: src/Components/GitApp/GitRepositories/Repository/index.js [hunk 1]
```diff
@@ -1,18 +1,32 @@
...
+      <Container>
+        <RepositoryName>{name}</RepositoryName>
+        <Wrapper>
+          <Stars>
+            <Icon name="star" size={20} />
+            <Text>Star {stars}</Text>
+          </Stars>
+          <Forks>
+            <Icon name="code-fork" size={20} />
+            <Text>Fork {forks}</Text>
```
Comment (0xc0d3r) [inline src/Components/GitApp/GitRepositories/Repository/index.js]: Integrate `react-native-localize` to setup I18n translations

File: src/Components/GitApp/GitRepositories/Repository/styledComponent.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  left: 30px;
+`;
+
+export const Wrapper = styled.View`
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+`;
+
+export const Stars = styled.View`
```
Comment (0xc0d3r) [inline src/Components/GitApp/GitRepositories/Repository/styledComponent.js]: Use flex

File: src/Components/GitApp/GitRepositories/Repository/styledComponent.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  position: absolute;
+  left: 30px;
+`;
+export const Forks = styled.View`
+  position: absolute;
```
Comment (0xc0d3r) [inline src/Components/GitApp/GitRepositories/Repository/styledComponent.js]: Use flex

File: src/Components/GitApp/GitUsers/User/styledComponent.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  border-bottom: 1px solid black;
+  margin-top: 10px;
+`;
+
+export const Text = styled.Text`
+  font-size: 25px;
+  position: absolute;
```
Comment (0xc0d3r) [inline src/Components/GitApp/GitUsers/User/styledComponent.js]: Use flex

File: src/Components/GitApp/GitUsers/User/styledComponent.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  left: 80px;
+`;
+
+export const Image = styled.Image`
+  height: 40px;
+  width: 40px;
+  border-radius: 50%;
+  position: absolute;
```
Comment (0xc0d3r) [inline src/Components/GitApp/GitUsers/User/styledComponent.js]: Use flex

File: src/Components/GitApp/GitUsers/index.js [hunk 6]
```diff
@@ -1,24 +1,40 @@
...
   }
 
   users = () => {
-    const { usersStore } = this.props;
-    const { users } = usersStore;
-    return this.users.map(user => (
-      <User user={user} gotoRepositoriesScreen={gotoRepositoriesScreen} />
-    ));
+    const { users, usersApiStatus } = usersStore;
+    if (usersApiStatus === apiStatus.LOADING) return <LoadingMsg />;
```
Comment (0xc0d3r) [inline src/Components/GitApp/GitUsers/index.js]: use `switch..case` instead of `if..else`

File: src/Components/GitApp/LoadingMsg/index.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
+import React, { Component } from "react";
+import { Text } from "react-native";
+
+import { Container } from "./styledComponent";
+
+class LoadingMsg extends Component {
+  render() {
+    return (
+      <Container>
+        <Text>Loading...</Text>
```
Comment (0xc0d3r) [inline src/Components/GitApp/LoadingMsg/index.js]: Use.. react native `ActivityIndicator`

File: src/stores/UsersStore/index.js [hunk 8]
```diff
@@ -1,30 +1,30 @@
...
+  @observable users = [];
+  @observable usersApiStatus;
   getUsers = async () => {
     try {
-      this.setReposApiStatus(apiStatus.LOADING)
-      let response = await fetch('https://api.github.com/users')
-      let responseJson = await response.json()
-      responseJson = responseJson.map(user => user)
+      this.setReposApiStatus(apiStatus.LOADING);
+      let response = await fetch("https://api.github.com/users");
```
Comment (0xc0d3r) [inline src/stores/UsersStore/index.js]: follow `services` & `fixtures` structure

File: src/stores/UsersStore/index.js [hunk 9]
```diff
@@ -1,30 +1,30 @@
...
+      let responseJson = await response.json();
+      responseJson = responseJson.map(user => user);
```
Comment (0xc0d3r) [inline src/stores/UsersStore/index.js]: why this `map` here?

File: package-lock.json [hunk 10]
```diff
@@ -0,0 +1,239 @@
+{
```
Comment (0xc0d3r) [inline package-lock.json]: Why are you using `npm` to install packages?

File: src/Components/GitApp/GitRepositories/Repository/styledComponent.js [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+`;
+
+export const Wrapper = styled.View`
+  display: flex;
+  flex-direction: row;
+`;
+
+export const Stars = styled.View`
+  margin-left: 10;
+  display: flex;
```
Comment (0xc0d3r) [inline src/Components/GitApp/GitRepositories/Repository/styledComponent.js]: there is no display flex in react native

File: src/Components/GitApp/GitRepositories/index.js [hunk 12]
```diff
@@ -1,8 +1,15 @@
...
-import { View } from "react-native";
+import { FlatList } from "react-native";
 
 import Repository from "./Repository";
+import LoadingMsg from "../LoadingMsg";
+import ErrorMsg from "../ErrorMsg";
 
+import { Container } from "./styledComponent";
+import { apiStatus } from "../../../constants";
+import { observer } from "mobx-react";
```
Comment (0xc0d3r) [inline src/Components/GitApp/GitRepositories/index.js]: follow clean code checklist `vertical separation` point

File: package-lock.json (hunk 10 above)
Comment (vamsimudadla) [reply package-lock.json]: again i used yarn add

**Pull request merged**